tests/integration/double_vote.go

```diff
@@ -307,12 +336,15 @@ func (s *CCVTestSuite) TestHandleConsumerDoubleVotingSlashesUndelegations() {
 		slashFraction := s.providerApp.GetTestSlashingKeeper().SlashFractionDoubleSign(s.providerCtx())
 
 		// check undelegations are slashed
-		ubds, _ := s.providerApp.GetTestStakingKeeper().GetUnbondingDelegation(s.providerCtx(), delAddr, validator.GetOperator())
+		ubds, _ = s.providerApp.GetTestStakingKeeper().GetUnbondingDelegation(s.providerCtx(), delAddr, validator.GetOperator())
 		s.Require().True(len(ubds.Entries) > 0)
 		for _, unb := range ubds.Entries {
 			initialBalance := sdk.NewDecFromInt(unb.InitialBalance)
 			currentBalance := sdk.NewDecFromInt(unb.Balance)
 			s.Require().True(initialBalance.Sub(initialBalance.Mul(slashFraction)).Equal(currentBalance))
 		}
+
```

Comment:
```suggestion
// check that redelegations are slashed
```

That's what the next 2 lines do, right?